src/AutoSplitImage.py

```diff
@@ -98,6 +121,39 @@ def __init__(self, path: str):
         else:
             self.image_type = ImageType.SPLIT
 
+    def __parse_text_file(self, path: str):
+        if not TESSERACT_PATH:
+            error_messages.tesseract_missing(path)
+            return
+
+        with open(path, encoding="utf-8") as f:
+            data = toml.load(f)
+
+        self.texts = [text.lower().strip() for text in data["texts"]]
+        self.__rect = [
+            data["top_left"][0],
+            data["bottom_right"][0],
+            data["top_left"][1],
+            data["bottom_right"][1],
+        ]
```

Comment:
Rect can be made a 4-length tuple. It would probably be even better as a NamedTuple, but I don't think we have any ourselves and I'd rather not depend on an external library just for a `Rect` named tuple (so that would come later)

```suggestion
        self.__rect = (
            data["top_left"][0],
            data["bottom_right"][0],
            data["top_left"][1],
            data["bottom_right"][1],
        )
```